usecases/collect_nodes_statuses.py

```diff
@@ -29,4 +30,11 @@ def _send(self, data: dict) -> None:
             self.log.warning("collect_status_error", error=data['error'])
             return
         node = Node.from_status_dict(data)
+        # XXX: a big enough known_peers list can reach the limit of data passed to a lambda.
+        # This filters the unnecessary peer ids.
+        old_known_peers = copy(node.known_peers)
```

Comment:
Do we need this copy? If it's not needed we should remove it so we don't use this amount of ram in the lambda